packages/editor/src/components/error-boundary/style.scss

```diff
@@ -1,5 +1,4 @@
 .editor-error-boundary {
-	max-width: $content-width;
```

Comment:
Please advise as to which `max-width` to keep. Currently preserving existing behavior, `$content-width` appears to be too narrow for this element (cf. #8834)

Reply:
> Please advise as to which `max-width` to keep. Currently preserving existing behavior, `$content-width` appears to be too narrow for this element (cf. #8834)

Since there's no history other than #8834 and the other style will have always taken effect, I assume you've made the correct assumption here.